performant-software/Annotation-Studio
# Allow users to access documents in their own anthologies

## What does this PR do?
1. Allow users to access documents in their own anthologies
2. Users can add or remove documents from their own anthologies

## How to test?
1. Sign in as a student user
2. Click create an anthology and create a sample anthology
3. Go to http://cove-staging.herokuapp.com/anthologies and click on the anthology you created
4. Search for and add a document to the anthology
5. Try to remove the document from the anthology
6. Add another document to the anthology
7. Click on the document you added to the anthology from the all tab
8. You should be able to access the document without any issues.
